### ISSUE TITLE
Self-Voicing, Paradigm 2 & 3 - Level 2 Example ready

### ISSUE THREAD

**Issue by terracoda** [phetsims/gravity-force-lab-basics#205]
From #202 we have now have two similar Paradigm's for a Layered Self-Voicing approach. 
Paradigm 2 and 3 essentially deliver the same content, though depending on the learner's input method, deliver it in different ways.

@jessegreenberg, I have a walk-through for adding Level 2 Context Responses ready for implementation.

Please see [Table P3 Level 2](https://docs.google.com/document/d/1XFTD2HTcGCEOP2yQ9_2SkEeXmCceZYUCmNeyiJfcCHs/edit#heading=h.vt9darjkrngu) in design doc 

There are 2 different versions - the Version 1 column uses customs strings already created for Level 0-1, and the Version 2 column uses strings from PDOM for the type of content that is delivered in Level 0-1. Both versions use the same PDOM context responses for adding in context changes.

Let me know if you have any questions.
I only did a walk through for P3 level 2. P2 is the same but with more objects in the focus order.

**Comment by jessegreenberg** [comment]
In my review of the work so far, it looks like the "Farther away" or "Closer" portion of the self voicing content is missing:
With Distance checked:
4.5 kilometers from mass 1. **Farther away**, force arrows get smaller, forces now 26.4 newtons.

**Comment by jessegreenberg** [comment]
It looks like the mass control is missing its help text on focus.

**Comment by jessegreenberg** [comment]
Everything requested has been added to the sim, and I went through the examples/walkthroughs and tried to catch any mistakes. Closing this issue since first pass is done and we are ready for review.
